cou/apps/app.py

```diff
@@ -118,6 +117,16 @@ def decorator(application: type[OpenStackApplication]) -> type[OpenStackApplicat
         return decorator
 
 
+@dataclass
+class ApplicationUnit:
+    """Representation of singe JujuUnit of application."""
+
+    unit: str
```

Comment:
not-blocking: Maybe name would be more suitable?

Reply:
any suggestions

Reply:
`name`

Reply:
+1 on renaming it to `name`

Reply:
done